docs/operations/testing.md

```diff
@@ -25,25 +50,33 @@ def test_query():
         }
     """
 
-    result = schema.execute_sync(
-        query,
-        variable_values={"title": "The Great Gatsby"},
-    )
+        self.schema = strawberry.Schema(Query)
+        self.result = self.schema.execute_sync(
+            self.query,
+            variable_values={"title": "The Great Gatsby"},
+        )
+
+    def test_sample_query(self):
+        assert self.result.errors is None
+        self.assertEqual(self.result.data["books"], [
+            {
+
+                "title": "The Great Gatsby",
+                "author": "F. Scott Fitzgerald",
+            }
+        ])
```

Comment:
I'd only create the schema inside the setup, or even outside the function. And move the query inside test_sample_query since it is specific for that test

```suggestion
        self.schema = strawberry.Schema(Query)

    def test_sample_query(self):
        query = ...
        result = self.schema.execute_sync(
            query,
            variable_values={"title": "The Great Gatsby"},
        )
        assert result.errors is None
        self.assertEqual(result.data["books"], [
            {

                "title": "The Great Gatsby",
                "author": "F. Scott Fitzgerald",
            }
        ])
```